cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -275,11 +276,22 @@ Then("{int} row/rows is/are visible", (int) => {
 });
 
 Then("I type {int} in pagination input", (value) => {
-  pageSelectDataComponent().find("input").type(value);
+  pageSelectInput().type(value, { force: true });
 });
 
 Then("Pagination input should have {int} value", (value) => {
-  pageSelectDataComponent().find("input").should("have.value", value);
+  pageSelectInput().should("have.value", value);
+});
+
+When("I press downarrow onto pagination select list", () => {
+  pageSelectInput().trigger("keydown", {
+    ...keyCode("downarrow"),
+    force: true,
```

Comment:
why we need to pass `force` here ?

Reply:
input is covered by a span element to hide the caret when hovering over the element, yet the input is still an element that is focused and accessed when using keyboard to be ARIA compatible